tdm-fe/src/app/page.tsx

```diff
@@ -46,19 +47,30 @@ const Home: React.FC = () => {
     const handleNext = async () => {
         const api = await DefaultApiFactory();
         switch (activeStep) {
-            case 0:
-                if (selectedWrapper?.fileType && file && selectedWrapper.fileType.indexOf(file.type) > -1) {
-                    setFileError(false);
-                    setLoading(true);
-                    api.traitmentUploadPost(file).then((res) => {
-                        setFileName(res.data.filename);
-                        setLoading(false);
-                        setActiveStep((prevActiveStep) => prevActiveStep + 1);
-                    });
-                } else {
+            case 0: {
+                if (!file || !selectedWrapper || !selectedWrapper.fileType) {
                     setFileError(true);
+                    break;
                 }
+
+                if (selectedWrapper.fileType.includes('application/x-gzip')) {
```

Comment:
<!-- Suggestions propose improvements to the current subject. It's important to be explicit and clear on what is being suggested and why it is an improvement. Consider using patches and the blocking or non-blocking decorations to further communicate your intent.  -->
**suggestion:** ‏Add a comment

```suggestion
                // For retro-compatibility with old web services
                if (selectedWrapper.fileType.includes('application/x-gzip')) {
```